src/components/views/rooms/EventTile.js

```diff
@@ -479,24 +481,31 @@ module.exports = WithMatrixClient(React.createClass({
         );
 
         var e2e;
+        let e2e_style;
```

Comment:
Hm. Changing the whole CSS to make room for 24h timestamps feels a bit worrying, although I see why you're doing it...

A nicer bet here would be to just put the `mx_EventTile_12h` class onto the EventTile itself, and then in the CSS have a selector like:

```
.mx_EventTile_12h .mx_EventTile_e2eIcon {
    padding-left: 5px;
}

.mx_EventTile_12h .mx_EventTile_line {
    padding-left: 100px;
}
```

...or whatever the tweaks are.  And then you don't need all these overrides of the e2e_style, which are quite confusing at first as it makes it sound like the e2e icon is somehow very explicitly being changed by the timestamp(!) rather than just for layout purposes.

TL;DR: cascading stylesheets cascade; let's make the most of that :)